packages/main/src/themes/sap_fiori_3_hcw/parameters-bundle.css

```diff
@@ -26,6 +26,7 @@
 @import "./MessageStrip-parameters.css";
 @import "./Panel-parameters.css";
 @import "../base/Popover-parameters.css";
+@import "./PopupsCommon-parameters.css";
 @import "../base/PopupsCommon-parameters.css";
```

Comment:
remove the below line that imports the base parameters file, because your file already does that in it.
same for packages/main/src/themes/sap_fiori_3_hcb/parameters-bundle.css